README.md

```diff
@@ -1,30 +1,133 @@
 # spanner-sql-udf
 
-This repository contains a set of Spanner User-Defined Functions (UDFs) for a
-number of MySQL built-in functions. Combined together with Spanner's existing
-built-in functions, they provide coverage for most of MySQL's built-in
-functions. These UDFs are designed to assist customers migrating applications
-from MySQL to Spanner by providing definitions for MySQL built-in functions not
-already supported in Spanner, reducing the effort to migrate application code
-and queries.
+This repository contains a set of Spanner User-Defined Functions (UDFs) for
+several MySQL built-in functions. Combined with Spanner's existing built-in
```

Comment:
How about:
This repository contains a package of Spanner User-Defined Functions (UDFs) for
many MySQL built-in functions. Combined with Spanner's existing built-in